### PR TITLE
Apply pyinstaller-action-linux fix

### PR DESCRIPTION
**Pull request by OldIMP**
Necessary changes after JackMcKew/pyinstaller-action-linux#1 is fixed

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -37,8 +37,8 @@ jobs:
     - name: Package Application
       uses: JackMcKew/pyinstaller-action-linux@main
       with:
-        path: /
+        path: .
     - uses: actions/upload-artifact@v2
       with:
         name: volksschule
-        path: dist/*.exe
+        path: dist/linux
```
Comment (OldIMP) [inline .github/workflows/python-app.yml:45]: @JackMcKew so I guess `dist/linux` is a hardcoded path in the docker image? BTW I get an [artifact w/o .exe extension](https://github.com/OldIMP/volksschule/suites/1029848268/artifacts/13615507), and when I run it, i got:
>This version of c:\Users\shouh\Downloads\r_quiz.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.

I'll create another issue for it.

Comment (JackMcKew) [reply .github/workflows/python-app.yml:45]: @OldIMP The `pyinstaller-action-linux` is specifically for packaging executables intended to run on linux.

Given the error your receiving, I take it that you're wanting to package executables intended for use on windows, which you should use https://github.com/JackMcKew/pyinstaller-action-windows instead for a much better experience (all features are the same)

Furthermore `dist/linux` and `dist/windows` are both hardcoded paths within the docker images so these technically can't change unless you're using a specific path which then it'd become `specific_path/dist/linux` or `specific_path/dist/windows`

Comment (OldIMP) [reply .github/workflows/python-app.yml:45]: Of course! My stupid! I'll change to pyinstaller-action-windows and let you know. Sry & thx!

Comment (JackMcKew) [reply .github/workflows/python-app.yml:45]: No problem 👍 I'll update the README to be more obvious

Comment (OldIMP) [reply .github/workflows/python-app.yml:45]: > @OldIMP The `pyinstaller-action-linux` is specifically for packaging executables intended to run on linux.
> 
> Given the error your receiving, I take it that you're wanting to package executables intended for use on windows, which you should use https://github.com/JackMcKew/pyinstaller-action-windows instead for a much better experience (all features are the same)
> 
> Furthermore `dist/linux` and `dist/windows` are both hardcoded paths within the docker images so these technically can't change unless you're using a specific path which then it'd become `specific_path/dist/linux` or `specific_path/dist/windows`

Thx @JackMcKew! This works perfectly! 👍 https://github.com/OldIMP/volksschule/commit/f8433b19577f9db7b087d3f61a7182bd9074b37b

Comment (JackMcKew) [reply .github/workflows/python-app.yml:45]: Fantastic!

**Pull request merged**